src/api/v2/ResponseHelper.js

```diff
@@ -39,20 +34,51 @@ const parseErrorObject = (errors) => {
  * e.g.  errors: [{ title: 'Invalid field', detail: 'tag_group is not a valid includable relationship of tags', code: '112', status: '400', },]
  * would return 'Invalid field tag_group is not a valid includable relationship of tags'
  */
-const parseErrorArray = (errors) =>
+const parseJsonApiErrorArray = (errors) =>
   errors.map(({ title, detail }) => `${title} ${detail}`).join(', ')
 
 /*
- * @param {Object} e.g. { success = true, body: { data: { id: 1 } }, errors = { error1: ['nasty'], error2: ['broken', 'crushed'], }, error = 'there was an error', error = { title: 'Invalid field', detail: 'tag_group is not a valid includable relationship of tags', code: '112', status: '400',
+ * @param {Array} errors
+ * @returns {String} - comma separated string of key, value pairs
+ * // e.g.  errors: [{ source: { pointer: '/data/attributes/printer', }, detail: "can't be blank", },] would return 'printer can't be blank'
+ */
+const parsePrintMyBarcodeErrors = (errors) => {
+  return errors
+    .map(({ source, detail }) => `${source.pointer.split('/').pop()} ${detail}`)
+    .join(', ')
+}
```

Comment:
Is there tests for this function?

Reply:
Yes. In ResponseHelper.spec.js 'failure with print my barcode request'